test/CMakeLists.txt

```diff
@@ -108,7 +109,7 @@ elseif(GLM_TEST_ENABLE_SIMD_AVX2)
 	add_definitions(-DGLM_FORCE_INTRINSICS)
 
 	if((CMAKE_CXX_COMPILER_ID MATCHES "GNU") OR (CMAKE_CXX_COMPILER_ID MATCHES "Clang"))
-		add_compile_options(-mavx2)
+		add_compile_options(-mavx2 -mfma)
```

Comment:
Ok, this is not ok, we need a new flags. I thought FMA was part of AVX2 instruction set but it's not.

Reply:
Indeed, FMA is not part of AVX2. It was introduce at the same time as AVX2, so most of Intel processor that use AVX2 are able to use FMA too. 
Not sure it's the case for AMD, so I will add a define. I didn't want to add one to prevent adding another behavior that should be test too. If you think it's mandatory, I will do.